### DISCUSSION TITLE
开启简化评论区，评论表情与右侧视频推荐起冲突

### DISCUSSION THREAD

**Discussion by noahhhi** [the1812/Bilibili-Evolved#3250] [综合讨论 / General]
![image](https://user-images.githubusercontent.com/29367133/163677327-676d124a-0a35-4570-b043-26b486edb13e.png)

**Comment by Geeyun-JY3** [comment]
请
善
用
搜
索
#2615、#3241

**Comment by noahhhi** [comment]
issue必填的太多，偷懒了
